src/test/java/testHW9/FlightBookingTest.java

```diff
@@ -0,0 +1,76 @@
+package testHW9;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class FlightBookingTest extends BaseTest {
+    private BaseTestAdmin baseTestAdmin;
+    //Create test data
+    String departureCity ="POS";
+    String destinationCity = "MYF";
+    String flightClass = "business";
+    int adultsNum = 2;
+    int childrenNum = 1;
+    int infantsNum = 1;
+
+    String firstName = "Oksana";
+    String lastName = "Skovron";
+    String email = "os@gmail.com";
+    String contactNumber = "3547877";
+    String address = "address 1 /7";
+    String country = "Spain";
+    String passengersName = "Passenger1";
+    String passengersAge = "25";
+    String passportNo = "ES58744";
+
+    @Test
+    public void testBooking(){
+        //Go to Flights tab, enter data and click Search
+        homePage.searchFlights(departureCity, destinationCity, flightClass, adultsNum, childrenNum, infantsNum);
+
+        //Assert that search results page has opened.
+        System.out.println(searchFlightResultsPage.modifySearchButtonGetText().toLowerCase());
+        Assert.assertTrue(searchFlightResultsPage.modifySearchButtonGetText().contains("modify search"),
```

Comment:
You've got a lot of assertions. Try to avoid asserting anything unless it's a part of test case's expected result.
If some page is not displayed - the test would fail either way.